core/src/main/java/hudson/slaves/Cloud.java

```diff
@@ -99,8 +101,16 @@ public String getDisplayName() {
         return name;
     }
 
-    public String getSearchUrl() {
-        return "cloud/"+name;
+    /**
+     * Get URL of the cloud.
+     *
+     * @since TODO
+     * @return Jenkins relative URL.
+     */
+    public @Nonnull String getUrl() { return "cloud/" + name; }
+
+    public @Nonnull String getSearchUrl() {
```

Comment:
Add Javadoc as well? Just to explain difference. Maybe it is also a subject for default implementation like @jglick did for job APIs